server/controller/Users/UserQueries.js

```diff
@@ -12,8 +12,8 @@ const mysql = require("mysql");
 module.exports = {
     register: (data, callBack) => {
         dbConfig.query(
-            `insert into users(username, password) 
-                        values(?,?)`,
+            `insert into users(username, password, userTypeID) 
+                        values(?,?,1)`,
```

Comment:
auto generate which mean that's random number @nguyenquocviet2007  please fix this!